tool/generate-app.sh

```diff
@@ -37,7 +37,7 @@ function export_mipmap_icon_all() {
   wait
 
   fvm dart run flutter_native_splash:create
-  fvm flutter gen-l10n
+  melos run generate:neon:l10n
```

Comment:
I think this doesn't even belong here in the first place. Generating all the app assets etc. is separate from generating l10n.